tracpro/templates/home/home.html

```diff
@@ -10,17 +10,75 @@
       {% trans "You don't have access to any regions. Contact your administrator." %}
     </div>
   {% else %}
-    <h2>
-        {% trans "Latest Baseline Data" %}
-      </h2>
-      <div class='row' style='margin-bottom: 1em' >
-        <a href='/baselineterm/read/{{ baselineterm.pk }}/'>
-          <div class='col-md-12'>
-            <div id="container" style="min-width: 210px; height: 300px; max-width: 800px; margin: 0 auto">
-            </div>
+    <div class='row' style='margin-bottom: 1em' >
+        <div class='col-md-6'>
+          <h2>
+            {% trans "Latest Baseline Chart" %}
+          </h2>
+          <div id="container" style="min-width: 210px; height: 300px; max-width: 800px; margin: 0 auto">
           </div>
-        </a>
-      </div>
+        </div>
+        <div class='col-md-6'>
+          <div class='page-header'>
+            {% if org_perms.baseline.baselineterm_list %}
+              <a class='btn btn-default pull-away' href='{% url 'baseline.baselineterm_list' %}'>
+                <span class='glyphicon glyphicon-th-list'></span>
+                {% trans "View All" %}
+              </a>
+            {% endif %}
+            <h2>
+              {% trans "Latest Baseline Terms" %}
+            </h2>
+          </div>
+          <table class='table table-striped' style='width: 100%'>
+          <thead>
+            <tr>
+              <th>
+                {% trans "Name" %}
+              </th>
+              <th>
+                {% trans "Start Date" %}
+              </th>
+              <th>
+                {% trans "End Date" %}
+              </th>
+              <th>
+                {% trans "Baseline Question" %}
+              </th>
+              <th>
+                {% trans "Follow Up Question" %}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for baselinetermitem in baselineterms %}
+              <tr>
+                <td>
+                  <a href='/baselineterm/read/{{ baselinetermitem.pk }}/'>
```

Comment:
This can be replaced with `{% url "baseline.baselineterm_read" baselinetermitem.pk %}` or, if appropriately hooked up, `{{ baselinetermitem.get_absolute_url }}`.